k8s/scalyr-agent-2-configmap.yaml

```diff
@@ -12,6 +12,7 @@ data:
   #
   # Example:
   # SCALYR_K8S_EVENTS_DISABLE: "true"
+  # SCALYR_K8S_RATELIMIT_CLUSTER_NUM_AGENTS: "1"
```

Comment:
WDYT about this?  Include or don't

Reply:
Don't.

I'm of the opinion that the rate limiter should stay hidden away until someone indicates they might need it.

Reply:
Agree 👍